concent_api/api_testing_common.py

```diff
@@ -0,0 +1,176 @@
+import datetime
+import http.client
+import json
+import requests
+import sys
+
+from collections                    import namedtuple
+
+from golem_messages.exceptions      import MessageError
+from golem_messages.message         import Message, ComputeTaskDef, TaskToCompute
+from golem_messages.shortcuts       import dump
+from golem_messages.shortcuts       import load
+
+
+ProtocolConstants = namedtuple("ProtocolConstants",
+                               ["concent_messaging_time",
+                                "subtask_verification_time",
+                                "force_acceptance_time",
+                                "token_expiration_time"])
+
+
+def get_protocol_constants(cluster_url):
+    url = f"{cluster_url}/api/v1/protocol-constants/"
+    response = requests.get(url)
+    json = response.json()
+    concent_messaging_time = json['concent_messaging_time']
```

Comment:
Please review styling in this file, I see all issues that were here on previous review.

Reply:
done